### PR TITLE
Updates from changelog: Website 

### PR DESCRIPTION
**Pull request by cwickham**
[Guide > Websites > Website Navigation](https://deploy-preview-968--mystifying-jepsen-fa4396.netlify.app/docs/websites/website-navigation.html): 

- [x] In website projects, a single sidebar with a `id` property will no longer be used as a global sidebar. It will instead be used as a sidebar for only pages which specify that `id` or pages linked to from the sidebar.
- [x] Add support for `navbar > toggle-position` to control whether the responsive navbar toggle appears on the right or the left.
- [x] Add support for setting `page-navigation: true|false` in either a page or in `_metadata.yml`. This allows individual pages or sections of a website to control whether `page-navigaation` appears.
- [x] Add support for `bread-crumbs: true|false` to control whether bread crumbs are displayed. Add support for display of breadcrumbs on full width (non-mobile) pages when `bread-crumbs` is true. Default value is true.
- [x] ([#5283](https://github.com/quarto-dev/quarto-cli/issues/5283)): Add support for setting `repo-actions: false` in a document to prevent the display of repository actions on a specific page.
- [x] ([#5964](https://github.com/quarto-dev/quarto-cli/issues/5964)): Add support for `repo-link-target` and `repo-link-rel` to control the corresponding attributes of repo-action links.

[Guide > Websites > Website Search](https://deploy-preview-968--mystifying-jepsen-fa4396.netlify.app/docs/websites/website-search.html):

- [x] Add support for `show-item-context` key within the `search` key to control whether page parents are display next to items in search results. Pass `tree`, `parent`, `root`, or boolean (if you pass true, `tree` is the default).
- [x] ([#5763](https://github.com/quarto-dev/quarto-cli/issues/5763)): Add support for a keyboard shortcut to launch search (defaults to `s`, `f` or `/`). Use `search` > `keyboard-shortcut` to override with your own key(s).
- [x] ([#7796](https://github.com/quarto-dev/quarto-cli/issues/7796)): Allow providing placeholder text using the language key `search-text-placeholder`

[Guide > Websites > Website Tools](https://deploy-preview-968--mystifying-jepsen-fa4396.netlify.app/docs/websites/website-tools.html):

- [x] ([#5389](https://github.com/quarto-dev/quarto-cli/issues/5389)): Allow a website project to provide a default image used in social metadata tags.
- [x] ([#5251](https://github.com/quarto-dev/quarto-cli/issues/5251)): Allow individual pages to specify `image: false` to prevent image discover for Twitter and Open Graph metadata.
- [x] ([#5624](https://github.com/quarto-dev/quarto-cli/issues/5624)): Add support for localized Cookie Consent (using either the document's language or by specifying the language explicitly under the cookie consent key).

### CODE REVIEW

File: docs/websites/website-tools.qmd [hunk 1]
```diff
@@ -198,7 +213,7 @@ In addition to this basic configuration, you can exercise more fine grained cont
 | `version`      | The version number of Google Analytics to use. Currently supports either 3 (for analytics.js) or 4 (for gtag). This is automatically detected based upon the `tracking-id`, but you may specify it. |
 +----------------+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 
-: {tbl-colwidths="[20,80]"}
+: {tbl-colwidths="\[20,80\]"}
```
Comment (dragonstyle) [inline docs/websites/website-tools.qmd:216]: Is this ok (I'm actually not sure)

File: docs/websites/website-tools.qmd [hunk 2]
```diff
@@ -250,12 +265,15 @@ You can further customize the appearance and behavior of the consent using the f
...
++--------------+--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 | `policy-url` | The url to the website's cookie or privacy policy.                                                                                                                                   |
 +--------------+--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 | `prefs-text` | The text to display for the cookie preferences link in the website footer.                                                                                                           |
 +--------------+--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 
-: {tbl-colwidths="[20,80]"}
+: {tbl-colwidths="\[20,80\]"}
```
Comment (dragonstyle) [inline docs/websites/website-tools.qmd:276]: Is this ok (I'm actually not sure)

**Review by dragonstyle — APPROVED** [review]
2 possible issues to look (though maybe completely fine). Otherwise this looks great!

File: docs/websites/website-tools.qmd (hunk 1 above)
Comment (cwickham) [reply docs/websites/website-tools.qmd:216]: 🤷🏻‍♀️ I think this is something the visual editor changed...I'll take a look and see if it matters.

Comment (cwickham) [reply docs/websites/website-tools.qmd]: Seems to be OK - at least it ends up giving the right `<col style="width: 20%">` HTML output. But no sense in changing what isn't broken...I'll revert.

**Pull request merged**